data/presets/office/courier.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "fas-people-carry-box",
+    "fields": [
+        "name",
+        "address",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "air_conditioning",
+        "branch_brand",
+        "brand"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "office": "courier"
+    },
+    "terms": [
+        "courier",
+        "delivery",
+        "express delivery",
+        "delivery service",
+        "parcel",
+        "package",
+        "messenger",
+        "carrier"
+    ],
```

Comment:
could you please sort these alphabetically?

Reply:
Fixed. Thank you for the feedback.